azext_iot/_help.py

```diff
@@ -851,7 +854,8 @@
                    While the device simulation is running, the device will automatically receive
                    and acknowledge cloud-to-device (c2d) messages. For mqtt simulation, all c2d messages will
                    be acknowledged with completion. For http simulation c2d acknowledgement is based on user
-                   selection which can be complete, reject or abandon.
+                   selection which can be complete, reject or abandon. Additionally, mqtt simulation is only
```

Comment:
Please verify if http simulation works with non-sas auth.

Reply:
Yes, I have verified that